src/component/variantPage/functionalPrediction/AlphaMissense.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { action, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { Collapse, Table } from 'react-bootstrap';
+import functionalImpactColor from './styles/functionalImpactTooltip.module.scss';
+import functionalGroupsStyle from '../functionalGroups.module.scss';
+
+export interface IAlphamissenseProps {
```

Comment:
```suggestion
export interface IAlphaMissenseProps {
```

Reply:
change it all the alphamissense to alphaMissense